### PR TITLE
doc: release: 3.3: Add notes about Atmel and UpdateHub

### PR DESCRIPTION
**Pull request by nandojve**
Add 3.3.0 release notes for Atmel and UpdateHub

Signed-off-by: Gerson Fernando Budke [nandojve@gmail.com](mailto:nandojve@gmail.com)

### CODE REVIEW

File: doc/releases/release-notes-3.3.rst [hunk 1]
```diff
@@ -574,8 +590,15 @@ Drivers and Sensors
...
 * Serial
 
+  * Atmel SAM: UART/USART: Added support to configure driver at runtime
+
 * SPI
 
+  * Atmel SAM:
+
+    - Added support to transfers using DMA.
+    - Added support to loopback mode for testing purposes.
```
Comment (stephanosio) [inline doc/releases/release-notes-3.3.rst:600]: Need to use `*`.

```suggestion
  * Atmel SAM:

    * Added support to transfers using DMA.
    * Added support to loopback mode for testing purposes.
```

File: doc/releases/release-notes-3.3.rst [hunk 2]
```diff
@@ -1150,6 +1183,11 @@ Libraries / Subsystems
 HALs
 ****
 
+* Atmel
+
+  * sam0: Added support for SAMC20/21.
+  * sam4l: Add ``US_MR_CHRL_{n}_BIT`` Register Aliases for USART Driver.
```
Comment (stephanosio) [inline doc/releases/release-notes-3.3.rst:1189]: Need to use the past tense.

```suggestion
  * sam4l: Added ``US_MR_CHRL_{n}_BIT`` Register Aliases for USART Driver.
```

**Review by stephanosio — DISMISSED** [review]
Looks ok to me overall.

There are some clean-ups needed, but I will do that during the final sweep.

**Review by nashif — DISMISSED** [review]

**Comment by stephanosio** [comment]
@nandojve This needs a rebase.

**Review by laurenmurphyx64 — DISMISSED** [review]

**Review by stephanosio — DISMISSED** [review]

**Comment by stephanosio** [comment]
@nandojve This needs another rebase :(

**Review by stephanosio — DISMISSED** [review]

**Review by stephanosio — DISMISSED** [review]

File: doc/releases/release-notes-3.3.rst [hunk 3]
```diff
@@ -1409,6 +1409,16 @@ Libraries / Subsystems
     buffer is insufficient has been fixed by making the default size 256 bytes
     instead of 64 when the shell MCUmgr transport is selected.
 
+  * UpdateHub:
+
+    * The integrity check was reworked to allow to use other libraries. Since
+      them UpdateHub uses mbedTLS library as default crypto library.
```
Comment (dkalowsk) [inline doc/releases/release-notes-3.3.rst:1415]: them/then

File: doc/releases/release-notes-3.3.rst [hunk 4]
```diff
@@ -1409,6 +1409,16 @@ Libraries / Subsystems
     buffer is insufficient has been fixed by making the default size 256 bytes
     instead of 64 when the shell MCUmgr transport is selected.
 
+  * UpdateHub:
+
+    * The integrity check was reworked to allow to use other libraries. Since
```
Comment (dkalowsk) [inline doc/releases/release-notes-3.3.rst:1414]: to allow use by other

**Review by dkalowsk — COMMENTED** [review]
Not usually one to nitpick too hard on these bits, but one stood out as needing a fix and the other just came along for the ride

File: doc/releases/release-notes-3.3.rst (hunk 4 above)
Comment (nandojve) [reply doc/releases/release-notes-3.3.rst]: Thanks!

**Review by stephanosio — APPROVED** [review]

**Review by henrikbrixandersen — APPROVED** [review]

**Pull request merged**